src/NuGet.Core/NuGet.Packaging/ContentModel/ContentPropertyDefinition.cs

```diff
@@ -144,6 +142,21 @@ public virtual bool TryLookup(string name, PatternTable table, out object value)
             return false;
         }
 
+        private static bool ContainsSlash(string name)
```

Comment:
Nope never filed anything; was when I was working exclusively upstream; before I got caught by the slippery slope and started contributing to coreclr